test/testrunner.py

```diff
@@ -167,7 +167,7 @@ def main():
             if test_type == "ec2":
                 pytest_cmd += ["--reruns=1", "--reruns-delay=10"]
             if is_pr_context():
-                pytest_cmd.append("--timeout=4860")
+                pytest_cmd.append("--timeout=5400")
```

Comment:
Can you explain the reason for this change? Setting timeout to 90 is dangerous because some of our builds timeout at 90, so our finalizers will not get properly executed this way. This number (81 mins) was used to account for any test setup time before pytest is executed -- will be adding a comment here in one of my PRs).

Reply:
Reverted.